spec/models/reading_spec.rb

```diff
@@ -41,5 +41,21 @@
       })
       expect(@reading.temp).to eq 57
     end
+
+    it "handles nil temperature properly", :vcr do
+      allow(WeatherMan).to receive(:outdoor_temp_for).and_return(nil)
+      user = create(:user, zip_code: '10216')
+      user.sensors.create(name: "abcdefgh")
+      params = ActionController::Parameters.new({
+        "reading" => {
+          "temp"=>67.42,
+          "verification"=>"c0ffee",
+          "time"=>1452241576.0,
```

Comment:
Separate every 3 digits in the integer portion of a number with underscores(_).<br>Surrounding space missing for operator `=>`.